service/src/main/java/tds/session/repositories/impl/SessionRepositoryImpl.java

```diff
@@ -40,7 +59,42 @@ public SessionRepositoryImpl(DataSource datasource) {
             sessionOptional = Optional.empty();
         }
 
-
         return sessionOptional;
     }
+
+    /**
+     * In order to preserve compatibility with the existing TDS system, this implementation executes an UPDATE against
+     * the existing record in the {@code session.session} table.  Once the new code has complete responsibility for
+     * managing a {@link Session}, this implementation will be updated to comply with the "immutability" database design
+     * concept.
+     *
+     * @param sessionId The id of the {@link Session} to pause
+     * @param newStatus The status of the {@link Session} is being paused
+     */
+    @Override
+    public void pause(final UUID sessionId, final String newStatus) {
```

Comment:
Oddly enough, I don't think the status is ever "paused"; I think it's almost always "closed".  Todd and I had a conversation earlier and agree that `pause` is probably the worst possible name for this method.  But I _do_ think there are status values besides "closed" that get passed to the legacy `pause` method.  I'll investigate further.